$OpenBSD: patch-src_od-fs_defines_h,v 1.1.1.1 2012/11/22 23:45:20 edd Exp $

We don't have a sys/ucontext.h so we can't support the JIT

--- src/od-fs/defines.h.orig	Thu Nov 22 08:57:57 2012
+++ src/od-fs/defines.h	Thu Nov 22 08:58:52 2012
@@ -93,8 +93,8 @@ extern FILE *g_fs_uae_sync_debug_file;
 #define FPU_UAE
 #define GFXFILTER
 
-#if defined(__x86_64__) || defined(__ppc__)
-// no JIT for these architectures
+#if defined(__x86_64__) || defined(__ppc__) || defined(OPENBSD)
+// no JIT for these architectures (or OpenBSD)
 #else
 #define JIT
 #define JIT_DEBUG
